.github/workflows/release-connect-npm-init.yml

```diff
@@ -36,15 +61,21 @@ jobs:
         run: yarn install
 
       - name: Build dependencies
-        # Running `build:libs` requires action to checkout with `submodules: true`.
         run: yarn build:libs
 
       - name: Run @trezor/connect create npm release branch
         run: |
-          git config --global user.name "trezor-ci"
-          git config --global user.email "${{ secrets.TREZOR_BOT_EMAIL }}"
-          gh config set prompt disabled
```

Comment:
@vdovhanych getting rid of "trezor-ci" git user here, since I think this was totally unnecessary.

Reply:
Ok